File: src/direccion.py
class Direccion:

    def __init__ (self, calle: str, ciudad: str, departamento: str, codigo_postal: str, pais: str):
        self.__calle = calle
        self.__ciudad = ciudad
        self.__departamento = departamento
        self.__codigo_postal = codigo_postal
        self.__pais = pais

# --- Getters ---
    def get_calle(self) -> str:
        return self.__calle

    def get_ciudad(self) -> str:
        return self.__ciudad

    def get_departamento(self) -> str:
        return self.__departamento

    def get_codigo_postal(self) -> str:
        return self.__codigo_postal

    def get_pais(self) -> str:
        return self.__pais

#Setters
    def set_calle(self, calle: str):
        self.__calle = calle

    def set_ciudad(self, ciudad: str):
        self.__ciudad = ciudad

    def set_departamento(self, departamento: str):
        self.__departamento = departamento

    def set_codigo_postal(self, codigo_postal: str):
        self.__codigo_postal = codigo_postal

    def set_pais(self, pais: str):
        self.__pais = pais

    def obtener_direccion_completa(self):
        return f"{self.__calle}, {self.__ciudad}, {self.__departamento}, {self.__codigo_postal}, {self.__pais}"

File: Televentas Java/Clases/Direccion.java
package Clases;

public class Direccion {

    private String calle;
    private String ciudad;
    private String departamento;
    private String codigoPostal;
    private String pais;

    public Direccion(String calle, String ciudad, String departamento,
                     String codigoPostal, String pais) {
        this.calle = calle;
        this.ciudad = ciudad;
        this.departamento = departamento;
        this.codigoPostal = codigoPostal;
        this.pais = pais;
    }

    // --- Getters ---
    public String getCalle() { return calle; }
    public String getCiudad() { return ciudad; }
    public String getDepartamento() { return departamento; }
    public String getCodigoPostal() { return codigoPostal; }
    public String getPais() { return pais; }

    // --- Setters ---
    public void setCalle(String calle) { this.calle = calle; }
    public void setCiudad(String ciudad) { this.ciudad = ciudad; }
    public void setDepartamento(String departamento) { this.departamento = departamento; }
    public void setCodigoPostal(String codigoPostal) { this.codigoPostal = codigoPostal; }
    public void setPais(String pais) { this.pais = pais; }

    public String obtenerDireccionCompleta() {
        return calle + ", " + ciudad + ", " + departamento + ", " + codigoPostal + ", " + pais;
    }
}
